cmd/manifest/main.go

```diff
@@ -72,4 +74,12 @@ func main() {
 	}
 
 	fmt.Println(string(b))
+
+	if *outputFile == "" {
+		return
+	}
+	err = os.WriteFile(*outputFile, b, 0664)
```

Comment:
nit: can this be squashed with the if below `if err := os.Write...; err != nil` etc.?